{"@t":"2019-07-29T07:45:06.6940109Z","@mt":"Error while running callback, remaining callbacks will not run.","@l":"Error","@x":"System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at Umbraco.Web.PublishedCache.NuCache.PublishedSnapshotService.<.ctor>b__22_6() in d:\\a\\1\\s\\src\\Umbraco.Web\\PublishedCache\\NuCache\\PublishedSnapshotService.cs:line 123\r\n   at Umbraco.Core.MainDom.OnSignal(String source) in d:\\a\\1\\s\\src\\Umbraco.Core\\MainDom.cs:line 134\r\n   at Umbraco.Core.MainDom.System.Web.Hosting.IRegisteredObject.Stop(Boolean immediate) in d:\\a\\1\\s\\src\\Umbraco.Core\\MainDom.cs:line 204\r\n   at System.Web.Hosting.HostingEnvironment.StopRegisteredObjects(Boolean immediate)","SourceContext":"Umbraco.Core.MainDom","ProcessId":19772,"ProcessName":"w3wp","ThreadId":228,"AppDomainId":47,"AppDomainAppId":"LMW3SVC7ROOT","MachineName":"DESKTOP-DHEDIOQ","Log4NetLevel":"ERROR"}
